### PR TITLE
Generate API docs from latest `develop-no`

### PR DESCRIPTION
**Pull request by ron-bierman-unity**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The valid JWT token must be supplied as part of the header to access API's all o
 
 <pre>curl --request GET \
   --url https://api-stg3.snapchance.no/profile \
-  --header 'authorization: Bearer $SNAPCHANCE_API_TOKEN`'</pre>
+  --header 'authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0eXBlIjoic2Vzc2lvbiIsImV4cCI6MTYxNzM0MzI2MywicGxheWVyX2lkIjoyLCJlbWFpbCI6IjU5Y2MifQ.fUyn-pli3fe6knErVCUj6WQ_5NbRR6_RMqtaB7Q21t4'</pre>
```
Comment (dkaga) [inline source/index.html.md]: Was this token meant to be updated to the hardcoded value?

**Review by dkaga — APPROVED** [review]

File: source/index.html.md [hunk 2]
```diff
@@ -496,185 +496,6 @@ To perform this operation, you must be authenticated by means of one of the foll
 bearerAuth
 </aside>
 
-## Onboard Completed
```
Comment (justinhamadeunity) [inline source/index.html.md]: @ron-bierman-unity why did this get removed?

Comment (dkaga) [reply source/index.html.md]: @justinhamadeunity I removed it in the swagger doc.  We don't use it anymore.

File: source/index.html.md (hunk 1 above)
Comment (ron-bierman-unity) [reply source/index.html.md]: Good catch.

The original idea was to have all example requests showing `$SNAPCHANCE_API_TOKEN`. 

Unfortunately it doesn't work. So this is how it's being generated from the current swagger file. So I think this change makes sense ATM.

Comment (dkaga) [reply source/index.html.md]: Ok thanks!

**Review by justinhamadeunity — APPROVED** [review]

**Pull request merged**